tests/servers/servers.go

```diff
@@ -278,7 +318,7 @@ func (server *HTTPServer) Start() {
 	http.HandleFunc("/config_headers_redirect", server.ConfigRedirect)
 	http.HandleFunc("/config_headers_redirected", server.ConfigRedirected)
 	http.HandleFunc("/config_headers_overwrite", server.ConfigHeadersOverwrite)
-
+	http.HandleFunc("/caBundles", server.CABundles)
```

Comment:
`Bundles` implies there's more than one bundle, which might be confusing.

Reply:
Yeah, I was under the impression that we might expect multiple cert bundles, so I named it accordingly.